specifyweb/frontend/js_src/lib/components/AttachmentsBulkImport/ViewAttachmentFiles.tsx

```diff
@@ -0,0 +1,214 @@
+              >
+                {file.parsedName?.toString()}
+              </Link.NewTab>
+            ) : (
+              resolvedRecord !== undefined && (
+                <div>
+                  {keyLocalizationMapAttachment[resolvedRecord.reason] ?? ''}
+                </div>
+              )
+            )}
+          </div>,
+        ],
+
+        attachmentId,
+        canDisambiguate: typeof handleDisambiguate === 'function',
+        isNativeError: resolvedRecord?.type === 'invalid',
+        isRuntimeError,
+      } as const;
+    }
+  );
+
+export function ViewAttachmentFiles({
+  uploadableFiles,
+  baseTableName,
+  uploadSpec,
+  onDisambiguation: handleDisambiguation,
+  onFilesDropped: handleFilesDropped,
+}: {
+  readonly uploadableFiles: RA<PartialUploadableFileSpec>;
+  readonly baseTableName: keyof Tables | undefined;
+  readonly uploadSpec: PartialAttachmentUploadSpec;
+  readonly onDisambiguation:
+    | ((
+        disambiguatedId: number,
+        indexToDisambiguate: number,
+        multiple: boolean
+      ) => void)
+    | undefined;
+  readonly onFilesDropped?: (file: FileList) => void;
+}): JSX.Element | null {
+  const [disambiguationIndex, setDisambiguationIndex] = React.useState<
+    number | undefined
+  >(undefined);
+
+  const data = React.useMemo(
+    () =>
+      resolveAttachmentDatasetData(
+        uploadableFiles,
+        setDisambiguationIndex,
+        baseTableName
+      ),
+    [uploadableFiles, setDisambiguationIndex, baseTableName]
+  );
+  const headers = React.useMemo(
+    () => ({
+      selectedFileName: commonText.selectedFileName(),
+      fileSize: attachmentsText.fileSize(),
+      record: (
+        <div className="flex min-w-fit items-center gap-2">
+          {baseTableName === undefined ? (
+            formsText.record()
+          ) : (
+            <>
+              <TableIcon label name={baseTableName} />
+              {uploadSpec.staticPathKey === undefined
+                ? ''
+                : defined(
+                    strictGetModel(baseTableName).getField(
+                      staticAttachmentImportPaths[uploadSpec.staticPathKey].path
+                    )
+                  ).label}
+            </>
+          )}
+        </div>
+      ),
+      status: attachmentsText.status(),
+      attachmentId: attachmentsText.attachmentID(),
+    }),
+    [uploadSpec.staticPathKey]
+  );
+
+  const fileDropDivRef = React.useRef<HTMLDivElement>(null);
+  const { isDragging, ...restCallbacks } = useDragDropFiles(
+    handleFilesDropped,
+    fileDropDivRef
+  );
+
+  const headerElements = React.useMemo(
+    () => Object.values(headers).map((label) => <>{label}</>),
+    [headers]
+  );
+
+  return data.length === 0 ? null : (
+    <>
+      <div
+        className="flex w-full flex-1 flex-col gap-2 overflow-auto rounded bg-[color:var(--background)] p-4 shadow-md"
+        {...restCallbacks}
```

Comment:
this container should be full size, and always rendered (not only if there are records - it's the table that should be rendered conditionaly) so that you can drag and drop files anywhere on the screen. at the moment, when you open the tool, since there is no table, you can only drag and drop onto the small button at the top - bad ux

Reply:
Reverted the change, fixed now!